templates/boundary-controller.hcl.j2

```diff
@@ -59,4 +59,43 @@ kms "gcpckms" {
   key_ring    = "{{ boundary_gcpckms_keyring }}"
   crypto_key  = "{{ boundary_gcpckms_key }}"
 }
-{% endif %}
\ No newline at end of file
+{% endif %}
+
+{% if boundary_kms_type == 'transit' %}
```

Comment:
I think we should change this to an elif and have it as a one if statement. Instead of having two separate IFs

Reply:
I changed to `elif`.